### PR TITLE
bisect: insort_*, unlike bisect_*, expects x to match type of elements of a

### PR DESCRIPTION
**Pull request by Kentzo**
See https://github.com/python/cpython/pull/20556/files#r432997024 for reference.

### CODE REVIEW

**Comment by Kentzo** [comment]
The only failure does not seem to relate to the change.

**Comment by AlexWaygood** [comment]
> The only failure does not seem to relate to the change.

If you close and reopen your PR, it should trigger a fresh CI run.

File: stdlib/_bisect.pyi [hunk 1]
```diff
@@ -27,27 +27,21 @@ if sys.version_info >= (3, 10):
         *,
         key: Callable[[_T], SupportsRichComparisonT] = ...,
     ) -> int: ...
-    @overload
-    def insort_left(a: MutableSequence[_T], x: _T, lo: int = ..., hi: int | None = ..., *, key: None = ...) -> None: ...
-    @overload
     def insort_left(
         a: MutableSequence[_T],
```
Comment (srittau) [inline stdlib/_bisect.pyi:31]: `_T` should probably be `SupportsRichComparisonT` here and elsewhere, but that's probably out of scope for this PR.

**Review by srittau — APPROVED** [review]
Thanks.

**Pull request merged**